tests/integration/data/FieldsTest.php

```diff
@@ -103,6 +103,7 @@ public function testSingleField() {
 			'conditions' => [$key => $id],
 			'fields' => [$key, 'name']
 		]);
+
```

Comment:
Accidentally adding a blank line. Files that are not functionally been changed, shouldn't appear in change sets.

Reply:
Thanks for your valuable feedback. I actually can't explain how the blank line sneaked in. I usually double-check the changes before committing and avoid whitespace-changes.